pegasus/helpers/page_helpers.rb

```diff
@@ -29,3 +29,12 @@ def partner_site?()
   partner_sites = CDO.partners.map{|x|x + '.code.org'}
   return partner_sites.include?(request.site)
 end
+
+def under_13?
+  under_13 = false
+  if dashboard_user
+    age = ((Date.today - dashboard_user[:birthday]) / 365).to_i
+    under_13 = true if age < 13
+  end
+  under_13
+end
```

Comment:
renaming it sounds good. I'm not sure if `current_user_under_13?` is 100% clear, but it will at least prompt the caller to look and see what it does. Alternately, if you were to invert the logic and call it `over_13_or_unknown?` then I think it'd be quite clear what it does.